### ISSUE TITLE
While moving the cursor in edit fields via the on-screen keyboard commands, TalkBack doesn't speak the character on the right of the cursor, as it should

### ISSUE THREAD

**Issue by GoogleCodeExporter** [2php/eyes-free#442]
```
What steps will reproduce the problem?
1. Use an on-screen keyboard witch has the option to move the text entry cursor 
via some method (e.g. the volume keys, with on-screen buttons, etc.).
2. Open a text field with some text in it and start moving the cursor via the 
keyboard's cursor moving method.

What is the expected output? What do you see instead?
While moving the text cursor in the text field, TalkBack doesn't announce the 
character on the right of the cursor, even though it should.

What version of the product are you using? On what operating system?
TalkBack 4.1.0 on a Samsung Galaxy Express GT-i8730 phone with Android 4.1.2 
(original Samsung ROM).

Please provide any additional information below.
This is a regression in TalkBack 4.1.0, because it works in TalkBack 3.5.2.
```

Original issue reported on code.google.com by `k.kolev1...@gmail.com` on 11 Mar 2015 at 9:18
